lib/python/picongpu/extra/input/createBunch_example.ipynb

```diff
@@ -230,16 +228,19 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# assigne to (existing) checkpoint file\n",
+    "# assign to (existing) checkpoint file\n",
     "# replace with your own paths and species name (from speciesDefinition.param of the input simulation)\n",
+    "# copyRNG can be used to skip the copying of the RNG values\n",
     "checkPoint_b = addParticles2Checkpoint(\n",
-    "    \"/bigdata/hplsim/production/wrobel45/PWFA-bunch-script_test/001_empty_test/simOutput/checkpoints/checkpoint_%T.bp\",\n",
-    "    \"/bigdata/hplsim/production/wrobel45/PWFA-bunch-script_test/002_bunch_test/simOutput/checkpoints/checkpoint_%06T.bp\",\n",
+    "    \"/bigdata/hplsim/production/wrobel45/PWFA_script/script_empty_021/simOutput/checkpoints/checkpoint_%T.bp5\",\n",
```

Comment:
Should we check in real paths? 
Could you clean them up to something not working but readable?